uw-frame-components/portal/widgets/partials/list-of-links.html

```diff
@@ -0,0 +1,34 @@
+<div class="widget-body list-of-links">
+
+  <!-- FOUR LINKS OR LESS -->
+  <div ng-show="config.links.length <= 4" class="list-of-links__{{config.links.length}}">
+    <circle-button ng-repeat="item in config.links"
+                   data-href="{{item.href}}"
+                   data-target="{{item.target}}"
```

Comment:
Could this be more secure if it included [`rel="noopener"`](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)?